tests/datasets/test_idtrees.py

```diff
@@ -140,15 +139,12 @@ def test_plot(self, dataset: IDTReeS) -> None:
             dataset.plot(x, show_titles=False)
             plt.close()
 
-    @pytest.mark.skipif(
-        sys.platform in ["darwin", "win32"],
-        reason="segmentation fault on macOS and windows",
-    )
     def test_plot_las(self, dataset: IDTReeS) -> None:
-        pytest.importorskip("open3d", minversion="0.11.2")
-        vis = dataset.plot_las(index=0, colormap="BrBG")
-        vis.close()
-        vis = dataset.plot_las(index=0, colormap=None)
-        vis.close()
-        vis = dataset.plot_las(index=1, colormap=None)
-        vis.close()
+        pytest.importorskip("pyvista", minversion="0.35.1")
+        import pyvista
+
+        # Test point cloud without colors
+        point_cloud = dataset.plot_las(index=0)
+        pyvista.plot(  # type: ignore[attr-defined]
```

Comment:
@banesullivan I assume type hints for pyvista are a work in progress? If you point me to where this function is defined I would be happy to add some.

Reply:
Indeed a work in progress though we do have partial: https://github.com/pyvista/pyvista/blob/main/pyvista/py.typed